bude_hezky/date/tomorrow_checker.py

```diff
@@ -0,0 +1,8 @@
+import datetime
+
+REAL_TOMORROW_DATE = datetime.date.today() + datetime.timedelta(days=1)
```

Comment:
Toto se vyhodnotí při "import time", tzn. tehdy kdy se ten soubor poprvé zparsuje a spustí Pythonem. Python není splachovací jako PHP, takže tam proces může běžet třeba týden a tahle proměnná by byla pořád nastavena na "zítra ode dne kdy jsem poprvé pustil tento Python program". Tudíž bych to přesunul spíš přímo do té funkce, aby se zítřek spočítal pokaždé, kdy se ta funkce spustí.